### PR TITLE
Fix GraphQL API to Sequelize

### PR DESCRIPTION
**Pull request by gamburgm**
Migrating to postgres killed GraphQL because the data in Elasticsearch no longer conformed to the API, that data needed to be fetched from the serializers. Migrating the API to use the serializers (and deleting bad code along the way).

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -335,8 +337,8 @@ async function onSendToMessengerButtonClick(sender, userPageId, b64ref) {
   macros.log('Got webhook - received ', userObject);
   // TODO: check that sender is a string and not a number
   const existingData = await database.get(sender);
-
-  const aClass = (await elastic.get(elastic.CLASS_INDEX, userObject.classHash)).class;
+  const classModel = await Course.findByPk(userObject.classHash);
+  const aClass = Object.values(await (new HydrateCourseSerializer(Section).bulkSerialize([classModel])))[0].class;
```
Comment (dajinchu) [inline backend/server.js:341]: Doesn't the serializer go fetch all the sections? Why fetch the sections if we're just throwing them away?

File: backend/graphql/resolvers/class.js [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
-  }
-};
+const serializer = new HydrateCourseSerializer(Section);
+
+const serializeValues = async (results) => {
+  return Object.values(await serializer.bulkSerialize(results));
+}
+
+const getLatestClassOccurrence = async (host, subject, classId) => {
+  const res = await Course.findOne({ where: { host, subject, classId } });
```
Comment (dajinchu) [inline backend/graphql/resolvers/class.js]: is this guaranteed to be the latest?

File: backend/graphql/resolvers/class.js [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
+  return (await serializeValues([res]))[0].class;
+}
+
+const getAllClassOccurrences = async (host, subject, classId) => {
+  const results = await Course.findAll({ where: { host, subject, classId }, limit: 10 });
```
Comment (dajinchu) [inline backend/graphql/resolvers/class.js]: are we confident a limit of 10 is enough? I think there may be more.

**Review by dajinchu — CHANGES_REQUESTED** [review]

File: backend/server.js (hunk 1 above)
Comment (gamburgm) [reply backend/server.js:341]: Yeah I'm 2 IQ

File: backend/graphql/resolvers/class.js (hunk 2 above)
Comment (gamburgm) [reply backend/graphql/resolvers/class.js]: Woops I had a sort in there at first but removed it while I was debugging.

File: backend/graphql/resolvers/class.js (hunk 3 above)
Comment (gamburgm) [reply backend/graphql/resolvers/class.js]: It had a limit of 10 before I added this, but the limit is pretty unnecessary, so I'll kill it.

**Review by dajinchu — APPROVED** [review]

**Pull request merged**